bbc/simorgh
# WSTEAMA-1110 - Add VJ Embed on Home and Topic pages

Resolves JIRA [[1110](https://jira.dev.bbc.co.uk/browse/WSTEAMA-1110)]

Overall changes
======
Allows an embed component to be rendered on Home and Topic pages when the embed html is present inside of the curation data.

Code changes
======

- Update to hindi homepage fixture data to include the vj curation.
- Update to Curation props to include an embed object.
- Embed object passed to Curation from Home page and Topic page.
- Adds a constant for `EMBED` inside of getComponentName and a conditional case added to return it when present.
- Test case added to cater for the presence of an `EMBED` component.

Testing
======
1. Checkout to the branch and `yarn dev` to spin up a local server.
2. Navigate to [http://localhost:7080/hindi](http://localhost:7080/hindi) and the election banner should render in the first position on the page. (or in the first position underneath an ad if you have ads switched on locally).

<img width="1438" alt="Screenshot 2024-03-21 at 12 39 49" src="https://github.com/bbc/simorgh/assets/120198886/564e48f7-5b39-481a-88e2-b305a3682591">


**Note**: Only home page testing notes are listed above, however I have tested the banner renders on a Topic page by inserting the embed curation into a topic fixture and spinning up a `yarn dev` to view it on the page. Future BFF work in an upcoming sprint will allow us to test this more throughly when Topic pages are able to support the same curations as Home pages.
